bundles/org.smarthomej.binding.androiddebugbridge/src/main/java/org/smarthomej/binding/androiddebugbridge/internal/AndroidDebugBridgeDevice.java

```diff
@@ -117,9 +121,18 @@ public void sendText(String text)
         runAdbShell("input", "text", URLEncoder.encode(text, StandardCharsets.UTF_8));
     }
 
```

Comment:
I have addressed this in to a separate pr, so you don't need to worry about it. I've also added a regex for the packageName to avoid passing weird strings to the command.